packages/jest-types/src/Config.ts

```diff
@@ -374,6 +378,8 @@ export type ProjectConfig = {
   extraGlobals: Array<keyof NodeJS.Global>;
   filter: Path | null | undefined;
   forceCoverageMatch: Array<Glob>;
+  freezeCoreModules: boolean;
```

Comment:
not quite sure if these should be on `ProjectConfig` or `GlobalConfig`. probably `GlobalConfig` 😵 . kindly assist.